src/main/java/bio/terra/cli/command/workspace/List.java

```diff
@@ -68,9 +70,9 @@ private void printText(java.util.List<UFWorkspaceLight> returnValue) {
 
   /** Column information for table output with `terra workspace list` */
   private enum Columns implements ColumnDefinition<UFWorkspaceLight> {
+    ID("ID", w -> w.id, 36, RIGHT),
```

Comment:
You might want to increase this column's width at the expense of the description or display name, since it's the ID we'll be using to do things.

Reply:
My thinking is that most IDs will be less than 36, and won't be cut off.

Right now terminal has to be 145 characters wide (otherwise we wrap):

![image](https://user-images.githubusercontent.com/10929390/167226600-18519bbb-5c80-471e-ad9b-b74915f16b78.png)

Maybe we should make Description shorter?